---
version: "1.0"

rs_config:
  enable_pointcloud:
    default: "true"
    description: "enable pointcloud"
  enable_fisheye1:
    default: "false"
    description: "enable fisheye 1"
  enable_fisheye2:
    default: "false"
    description: "enable fisheye 2"

d435i_R:
  serial: 
    data: "109622070324"
    dtype: "string"
  rotation_base_to_link:
    rows: 3
    cols: 3
    data: [
       0.7716, -0.5000, -0.3932,
       0.4540,  0.0000,  0.8910,
      -0.4455, -0.8660,  0.2270,
    ]
    dtype: "float"
  ZYX_base_to_link:
    rows: 3
    cols: 1
    data: [0.5318,0.4617,-1.3144]
    dtype: "float"
  translation_base_to_link:
    rows: 3
    cols: 1
    data: [0.3843,-0.02048,-0.0453]
    dtype: "float"

d435i_L:
  serial: 
    data: "112222071586"
    dtype: "string"
  dtype: string
  rotation_base_to_link:
    rows: 3
    cols: 3
    data: [
       0.7716, -0.5000,  0.3932,
      -0.4540,  0.0000,  0.8910,
      -0.4455, -0.8660, -0.2270,
    ]
    dtype: "float"
  ZYX_base_to_link:
    rows: 3
    cols: 1
    data: [-0.5318,0.4617,-1.8272]
    dtype: "float"
  translation_base_to_link:
    rows: 3
    cols: 1
    data: [0.3843,0.02048,-0.0453]
    dtype: "float"